vue/src/views/GeneralSettings.vue

```diff
@@ -30,7 +30,11 @@
 				v-model="startCalls"
 				:options="startCallOptions"
 				label="label"
-				track-by="value" />
+				track-by="value"
+				@input="saveChanges" />
+		</p>
+		<p>
+			<em>{{ t('spreed', 'When a guest or user can not start a call they can still join the conversation and chat. When a user or moderator started the call they can also join the call.') }}</em>
```

Comment:
The description is a bit confusing. I would say something like "When a call has started, everyone who has access to the conversation will be able to join the call."

Reply:
I wanted to make it clear that they can still chat, so people dont get it confused with the lobby thing